toolbox/README.md

```diff
@@ -1,7 +1,45 @@
 #  Rook Toolbox
 The rook toolbox is a container with common tools used for rook debugging and testing.  All packages in the toolbox can be seen in the [Dockerfile](Dockerfile)
 
-## Container Linux by CoreOS
+## Installing more tools
+The rook toolbox is based on Ubuntu, so more tools of your choosing can be easily installed with `apt-get`.  For example, to install `telnet`:
+```
+apt-get update
+apt-get install telnet
+```
+
+## Running the toolbox
+### Kubernetes
+The rook toolbox can run as a pod in a Kubernetes cluster.  First, ensure you have a running Kubernetes cluster with rook deployed (instructions can be found in the [Rook on Kubernetes readme](../demo/kubernetes/README.md)).
```

Comment:
yes, we can update this link along with the moving of k8s walkthrough to main readme